src/main/scala/org/broadinstitute/dsde/rawls/model/WorkspaceModel.scala

```diff
@@ -158,7 +129,7 @@ object AttributeConversions {
     case AttributeBoolean(b) => b
     case AttributeNumber(n) => n.bigDecimal
     case AttributeString(s) => s
-    case AttributeValueList(l) => l.map(attributeToProperty(_)).asJava
```

Comment:
It got moved to be reused... by expression stuff it looks like, not part of this PR